src/docs/authentication.yaml

```diff
@@ -91,6 +91,41 @@ paths:
           description: 'Bad request (either validation or database error)'
         '500':
           description: 'Internal server error'
+  /api/auth/login:
```

Comment:
I already changed this and merged it to the dev branch.

Reply:
No problem. It had flaws. It'll be fixed with this PR.